Edge/FIX_PERMISSIONS.cmd

```diff
@@ -0,0 +1,3 @@
+REM Grant necessary fs permissions to the Edge sandbox user to allow access to the extension's directory
+cd /d %~dp0
+icacls "." /grant "*S-1-15-2-3624051433-2125758914-1423191267-1740899205-1073925389-3782572162-737981194":"(OI)(CI)(WDAC,WO,GE)" 
```

Comment:
All the edge extensions Microsoft released do it in their install scripts. Turn-Off-the-Lights has it too 
https://github.com/turnoffthelights/Turn-Off-the-Lights-Edge-extension/blob/master/Setup.cmd